php/laravel/container/Application/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM php:7
 RUN apt-get update -y && apt-get install -y openssl zip unzip git
 RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
-RUN docker-php-ext-install pdo mbstring
```

Comment:
is this extension not needed?

Reply:
I verified all php scenarios after removing this. All are working fine. Not sure, why was it added in the first place.